ratelimit/redis_test.go

```diff
@@ -377,7 +376,7 @@ func TestFailOpenOnRedisError(t *testing.T) {
 		settings.Group,
 	)
 
-	allow := c.AllowContext(context.Background(), "akey")
```

Comment:
Lets revert this change, Allow() is kind of deprecated actually

Reply:
It's used in all tests in this file actually

Reply:
Then lets update tests - ratelimit filters call AllowContext